### PR TITLE
Add VolumePerLength yd3 / ft and ftUS

### PR DESCRIPTION
**Pull request by gojanpaolo**
Succeeds #733

### CODE REVIEW

File: UnitsNet.Tests/CustomCode/VolumePerLengthTests.cs [hunk 1]
```diff
@@ -32,5 +32,9 @@ public class VolumePerLengthTests : VolumePerLengthTestsBase
         protected override double LitersPerMeterInOneCubicMeterPerMeter => 1000;
 
         protected override double OilBarrelsPerFootInOneCubicMeterPerMeter => 1.917134088;
+
+        protected override double CubicYardsPerFootInOneCubicMeterPerMeter => 0.3986633487670267521605;
```
Comment (angularsen) [inline UnitsNet.Tests/CustomCode/VolumePerLengthTests.cs]: I was not able to verify these as I could not find any references, but I trust you checked it.

**Comment by angularsen** [comment]
[Release UnitsNet/4.43.0 · angularsen/UnitsNet](https://github.com/angularsen/UnitsNet/releases/tag/UnitsNet%2F4.43.0)

File: UnitsNet.Tests/CustomCode/VolumePerLengthTests.cs (hunk 1 above)
Comment (gojanpaolo) [reply UnitsNet.Tests/CustomCode/VolumePerLengthTests.cs]: A quick check would be
```
1 m² = 1.19599 yd²
1.19599 yd² * (1 yd / 3 ft) = 0.3986633 yd³ / ft
```

**Comment by gojanpaolo** [comment]
Thank you!

**Pull request merged**